Jenkinsfile

```diff
@@ -15,7 +15,7 @@ node {
 
     stage('Installing Packages') {
       sh("rm -rf venv")
-      sh("virtualenv -p python3 --no-site-packages venv")
+      sh("virtualenv -p python3.5 --no-site-packages venv")
```

Comment:
If installing Python 3 changes the `python3` binary to point to `python3.5` rather than `python3.4` I don't think we'll need this change.

Reply:
I found `python3` still returns the version `3.4` but specifying `3.5` worked!

Reply:
Ah I wonder if `3.4` ends up staying around because it's a dependency of some other Ubuntu package even if we don't explicitly install it...